xml/System.IO.Pipelines/PipeReader.xml

```diff
@@ -97,8 +98,12 @@ The memory for the consumed data will be released and no longer available.
 
 The <xref:System.IO.Pipelines.ReadResult.Buffer> previously returned from <xref:System.IO.Pipelines.PipeReader.ReadAsync(System.Threading.CancellationToken)> must not be accessed after this call.
 
+This call is equivalent to calling <xref:System.IO.Pipelines.AdvanceTo(System.SequencePosition, System.SequencePosition)> with the examined position same as the consumed position.
 The examined data communicates to the pipeline when it should signal more data is available.
 
+The consumed parameter should be greater than or equal to the examined position in the previous call to AdavanceTo. Otherwise <xref:System.InalidOperationException> will be thrown.
```

Comment:
Nit:

```suggestion
Because the consumed parameter doubles as the examined parameter, the consumed parameter should be greater than or equal to the examined position in the previous call to AdavanceTo. Otherwise <xref:System.InalidOperationException> will be thrown.
```

Reply:
Again the corrections make it more concise and clear